pkgs/by-name/ni/nix-fast-build/package.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  nix-eval-jobs,
+  nix-output-monitor,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nix-fast-build";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Mic92";
+    repo = "nix-fast-build";
+    rev = "refs/tags/${version}";
+    hash = "sha256-8zW6eWvE9T03cMpo/hY8RRZIsSCfs1zmsJOkEZzuYwM=";
+  };
+
+  build-system = [ python3Packages.setuptools ];
+
+  makeWrapperArgs = [
+    "--prefix PATH : ${
+      lib.makeBinPath [
+        nix-eval-jobs
+        nix-eval-jobs.nix
+        nix-output-monitor
+      ]
```

This is for riscv64 support:

```suggestion
      ]  ++ lib.optional (lib.meta.availableOn stdenv.hostPlatform nix-output-monitor.compiler) nix-output-monitor
```